src/components/Navbar.js

```diff
@@ -152,7 +166,10 @@ function Navbar() {
             <Link href={item.url}>{item.title}</Link>
           </MenuItem>
         ))}
-        <MenuItem onClick={handleClose} sx={{ paddingLeft: '10px' }}>
+        <MenuItem
+          onClick={handleClose}
+          sx={{ paddingLeft: '10px', display: mobile }}
```

Comment:
Instead of add mutiple `display` how about we just simply remove the whole component if condition mathes?

```
if(path === true) return null;

Reply:
Okay sure I'll do this